buildspecs/linux_arm_linaro.spec

```diff
@@ -0,0 +1,286 @@
+		<property name="graph_property.linux_2.6.graph_req.aux1" value="{$machine_mapping.2.6$}"/>
+		<property name="graph_property.linux_2.6.graph_req.machine.test" value="{$machine_mapping.arm$}"/>
+		<property name="graph_property.linux_2.6.graph_variant.testing_suffix" value=" for linux 2.6"/>
+		<property name="graph_property.linux_2.6.graph_variant.trailingID" value=".linux_386_2.6"/>
+		<property name="graph_property.linux_hammer.graph_req.aux1" value="{$machine_mapping.hammer$}"/>
+		<property name="graph_property.linux_hammer.graph_req.machine.test" value=""/>
+		<property name="graph_property.linux_hammer.graph_variant.testing_suffix" value=" for linux hammer"/>
+		<property name="graph_property.linux_hammer.graph_variant.trailingID" value=".linux_hammer"/>
+		<property name="graph_req.arch0" value="arch:arm"/>
+		<property name="graph_req.arch1" value="arch:32bit"/>
+		<property name="graph_req.aux0" value=""/>
+		<property name="graph_req.build" value="{$common.req.build.java9$}"/>
+		<property name="graph_req.build2" value="{$common.req.build.java8$}"/>
+		<property name="graph_req.machine" value="{$machine_mapping.x86$}"/>
+		<property name="graph_req.os" value="{$machine_mapping.linux$}"/>
+		<property name="graph_req.os.build" value="{$spec.property.graph_req.os$}"/>
+		<property name="graph_req.os.perf" value="{$spec.property.graph_req.os$}"/>
+		<property name="graph_se_classlib.java5" value="jcl_se.zip"/>
+		<property name="graph_se_classlib.java6" value="jcl_se.zip"/>
+		<property name="graph_variants" value="linux_2.4,linux_2.6,linux_hammer"/>
+		<property name="isReallyUnix" value="true"/>
+		<property name="j2seRuntimeDir" value="jre/lib/arm"/>
+		<property name="j2seTags" value="pxr3260,j9vmxr3224"/>
+		<property name="j9BuildName" value="linux_arm"/>
+		<property name="j9dt.compileTarget" value="makefile"/>
+		<property name="j9dt.make" value="gmake"/>
+		<property name="j9dt.toolsTarget" value="buildtools.mk"/>
+		<property name="javatestPlatform" value="linux_arm-32"/>
+		<property name="jclMaxSecurityPolicyDefine" value=" &quot;-Djava.security.policy=http://jcl1.ottawa.ibm.com/testres/java.policy&quot;"/>
+		<property name="jclMemoryMax" value="-Xmx32m"/>
+		<property name="jclOSStackSizeMax" value=""/>
+		<property name="jdiTestsSupported" value="true"/>
+		<property name="jgrinderTestingSupported" value="true"/>
+		<property name="jitTestingOptLevel" value="optlevel=warm"/>
+		<property name="localRootPath" value="$(J9_UNIX_ROOT)"/>
+		<property name="longLimitCmd" value=""/>
+		<property name="main_shortname" value="xr32"/>
+		<property name="os.lineDelimiter" value="unix"/>
+		<property name="platform_arch" value="arm"/>
+		<property name="sun.jdk7.platform_id" value="linux-arm"/>
+		<property name="sun.jdk8.platform_id" value="linux-arm"/>
+		<property name="svn_stream" value=""/>
+		<!-- cross compiler path not needed with Docker image -->
```

Comment:
Its the environment setup that obsoletes the need for the cross compiler path, not Docker.